src/site/content/en/learn/css/backgrounds/index.md

```diff
@@ -0,0 +1,456 @@
+---
+title: Backgrounds
+description: >
+  In this module learn the ways you can style backgrounds of CSS boxes using CSS
+audio:
+  title: The CSS Podcast - 053: Background
+  src: https://traffic.libsyn.com/secure/thecsspodcast/TCP052_v1.mp3?dest-id=1891556
+  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+authors:
+ - lozandier
+date: 2021-9-15
+
+---
+
+# Backgrounds
+
+Behind every CSS box is a specialized layer behind the content called the background layer. CSS provides a variety of ways to make meaningful changes to it–including allowing multiple layers.
+
+This module overviews the ways CSS authors can control the styles of this space.
+
+Background layers are furthest from the user, rendered behind the contents of a CSS box starting from its `padding-box` region. This enables the background layer to not overlap with borders at all.
+
+{% Codepen {
+  user: 'argyleink',
+  id: 'BaLedvd',
+  tab: 'css,result'
+} %}
+
+By default, the background layer is directly behind the `border-box` region of a CSS box, a border specified for a CSS box is therefore rendered on top of the background layer, even when a background layer is extended to cover the `border-box`.
+
+## Background Color
+
+One of the simplest effects you can apply to a background layer is set a [valid CSS color](). The initial value of `background-color` is `transparent`, which allows the contents of a parent to be seen through a CSS box.
+
+A color set on a background layer sits behind other things painted on a CSS box. This includes borders and outlines (editor’s note: Will deliberately set background-origin to border-box). The demo below demonstrates this.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'XWgRRMQ',
+  tab: 'css,result'
+} %}
+
+A `background-color` is also guaranteed to sit behind other layers you can create with background-related CSS properties such as a background-image. More on that next.
+
+## Background Image
+
+On top of the `background-color` is a background image, added with the `background-image` property. A `background-image` accepts the following:
+
+* A URL or data URLI using the `url` CSS function.
+* An image dynamically created by a gradient CSS function.
+
+### Setting a background-image with the `url` CSS function.
+
+The `url()` CSS  function enables CSS authors to reference a resource as an absolute URL, relative URL, or embed a resource itself in a CSS document by referencing the resource as a  data URI.
+
+The following demo demonstrates this:
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'JjJNNro',
+  tab: 'css,result'
+} %}
+
+{% Aside %}
```

Comment:
I don't think you close this Aside anywhere. Add `{% endAside %}` when it's done.